crates/forge/tests/cli/test_cmd/mod.rs

```diff
@@ -4206,3 +4207,111 @@ Tip: Run `forge test --rerun` to retry only the 2 failed tests
 
 "#]]);
 });
+
+forgetest_init!(should_fuzz_literals, |prj, cmd| {
```

Comment:
that's awesome! I assume this one should pass now quite quickly and we can include too? https://github.com/foundry-rs/foundry/blob/e98bff5a57d9f23e60b96e23627512eb5397a598/crates/forge/tests/cli/test_cmd/invariant/storage.rs#L3-L6

Since `should_fuzz_literals` test is only for stateless fuzz, would it make sense to add an invariant one as well? I am thinking in particular to replicate this one https://github.com/foundry-rs/foundry/blob/e98bff5a57d9f23e60b96e23627512eb5397a598/crates/forge/tests/cli/test_cmd/invariant/common.rs#L440 which we weren't able to break without providing the fixtures hints.